packages/demo/webpack.config.js

```diff
@@ -8,6 +8,9 @@ const prod = mode === 'production'
 module.exports = {
   entry: './src/main.js',
   resolve: {
+    fallback: {
```

Comment:
Oh good catch!  This would be good to add to the Core README for webpack and docs site!

Reply:
Nice good idea!